trade-finance-manager-ui/server/controllers/case/amendments/amendmentIsUsingFacilityEndDate.controller.js

```diff
@@ -0,0 +1,82 @@
+const { isTfmFacilityEndDateFeatureFlagEnabled } = require('@ukef/dtfs2-common');
+const api = require('../../../api');
+const { AMENDMENT_STATUS } = require('../../../constants/amendments');
+const { isUsingFacilityEndDateValidation } = require('./validation/amendmentIsUsingFacilityEndDate.validate');
+
+const getAmendmentIsUsingFacilityEndDate = async (req, res) => {
+  const { facilityId, amendmentId } = req.params;
+  const { userToken } = req.session;
+  const { data: amendment, status } = await api.getAmendmentById(facilityId, amendmentId, userToken);
+
+  if (status !== 200) {
+    return res.redirect('/not-found');
+  }
+
+  if (!amendment.changeCoverEndDate || !isTfmFacilityEndDateFeatureFlagEnabled()) {
+    return res.redirect(`/case/${amendment.dealId}/facility/${facilityId}/amendment/${amendmentId}/amendment-options`);
+  }
+
+  const isEditable = amendment.status === AMENDMENT_STATUS.IN_PROGRESS && amendment.changeCoverEndDate && isTfmFacilityEndDateFeatureFlagEnabled();
```

Comment:
I think the latter things are now redundant because of the redirect above
```
  if (!amendment.changeCoverEndDate || !isTfmFacilityEndDateFeatureFlagEnabled()) {
```
(If i can negate statements correctly)

Reply:
Good spot, have changed, thanks!